src/ConfigAggregator.php

```diff
@@ -92,6 +96,34 @@ public function getMergedConfig()
         return $this->config;
     }
 
+    /**
+     * Throw an exception if duplicate config providers are found.
+     *
+     * @param ProviderIterable $providers
+     * @return ProviderIterable
+     * @throws InvalidConfigProviderException
+     */
+    private function validateDuplicateProviders(iterable $providers): iterable
```

Comment:
I think it would be better as
```suggestion
    private function validateNoDuplicateProviders(iterable $providers): iterable
```

Reply:
Modified.